.github/workflows/test.yml

```diff
@@ -12,6 +12,11 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - name: setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
```

Comment:
Let's change this to '7.2' which is what the [Docker container that runs the tests](https://github.com/localgovdrupal/drupal-container/blob/596e0bc03241c240d1f7bf05ea5c53ba55a233ac/Dockerfile#L17) uses.

Reply:
Yes, I just tried that, so seeing how that goes in one of my tests.